Tests/Resources/Kernel/Lti1p3TestKernel.php

```diff
@@ -35,13 +35,17 @@ class Lti1p3TestKernel extends Kernel
 {
     use MicroKernelTrait;
 
-    public function registerBundles()
+    public function registerBundles(): iterable
     {
-        return [
+        $bundles = [
             new FrameworkBundle(),
             new SecurityBundle(),
             new Lti1p3Bundle()
```

Comment:
probably it should something like this for useful usage with generator below
```suggestion
            FrameworkBundle::class,
            SecurityBundle::class,
            Lti1p3Bundle::class
```